internal/kubernetes/kubernetes.go

```diff
@@ -596,13 +596,22 @@ func (k *Kubernetes) ClusterRoles() ([]string, error) {
 		return nil, err
 	}
 
-	clusterRoles, err := clientset.RbacV1().ClusterRoles().List(context.Background(), metav1.ListOptions{LabelSelector: "kubernetes.io/bootstrapping=rbac-defaults"})
+	rbacDefaults, err := clientset.RbacV1().ClusterRoles().List(context.Background(), metav1.ListOptions{
```

Comment:
Is there a way to combine these into one query?

Reply:
Not that I'm aware. The value is taken as a union and I didn't see an OR operator